app/editor/src/hooks/api-editor/interfaces/IUserFilter.ts

```diff
@@ -8,5 +6,8 @@ export interface IUserFilter {
   lastName?: string;
   isEnabled?: boolean;
   isSystemAccount?: boolean;
-  status?: UserStatusName;
+  status?: string;
+  sort: any;
+  role?: string;
```

Comment:
This interface is representing what the API is expected.  The API is expected the enum `status` values.  `sort` should follow the same structure you can see in the `IContentFilter`.  `role` doesn't exist in the API, it should be `roles` and an array of string.